boards/nucleo-l073/include/periph_conf.h

```diff
@@ -214,14 +214,14 @@ static const spi_conf_t spi_config[] = {
 #define I2C_1_EVT_IRQ       I2C3_IRQn
 #define I2C_1_EVT_ISR       isr_i2c3
 /* I2C 1 pin configuration */
-#define I2C_1_SCL_PORT      PORT_A
-#define I2C_1_SCL_PIN       8
-#define I2C_1_SCL_AF        3
-#define I2C_1_SCL_CLKEN()   (periph_clk_en(AHB, RCC_IOPENR_GPIOAEN))
-#define I2C_1_SDA_PORT      PORT_B
-#define I2C_1_SDA_PIN       5
-#define I2C_1_SDA_AF        8
-#define I2C_1_SDA_CLKEN()   (periph_clk_en(AHB, RCC_IOPENR_GPIOBEN))
+#define I2C_1_SCL_PORT      PORT_C
```

Comment:
At the begening, i just wanted to used I2C3 and no I2C2 but i have finally added the two channels. So, as you noticed, i have used PB13/14 for I2C2 and kept PC0/PC1 for I2C3. Now, it is possible to configure the 3 channels on L0 family.